pkg/store/mastership/store.go

```diff
@@ -37,6 +39,9 @@ type Store interface {
 	// IsMaster returns a boolean indicating whether the local node is the master for the given device
 	IsMaster(id topodevice.ID) (bool, error)
 
+	// GetTerm returns the mastership term for a given device
+	GetTerm(id topodevice.ID) (Term, error)
```

Comment:
We might as well make this `GetMastership` instead of `GetTerm` so all the relevant information is returned in one step. Otherwise, mastership can change between `IsMaster` and `GetTerm`

Reply:
Yeah. I was thinking about it... IsMaster function performs election as well which is a bit confusing. Suppose we have device1 and device2. If we call IsMaster for device1 then a master will be selected if it doesn't exist. So GetMastership performs election and returns all related information including Term assuming if a master doesn't exist for a device it picks one.